tooling/nargo_cli/tests/execution_success/simple_shield/src/main.nr

```diff
@@ -13,7 +13,8 @@ fn main(
   // Receiver public key
   to_pubkey_x: Field, 
   to_pubkey_y: Field, 
-) -> pub [Field; 2] {
+) -> pub [Field; 2]
+{
```

Comment:
This seems like a bit of an odd choice.

Reply:
I agree. Could we change this in the formatter?